maia-core/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
 name = "maia-core"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
❓ Is this not a breaking change? I'm assuming that anyone using `maia-core` will need to upgrade to `bdk:0.21`.